### DISCUSSION TITLE
Is it possible to make registry path of msedgewebview2.exe configurable?

### DISCUSSION THREAD

**Discussion by stevealex1** [MicrosoftEdge/WebView2Feedback#3989] [Q&A]
I want to configure WebView2 group policies from a custom path.

Is it possible to make registry path of msedgewebview2.exe configurable?
ie. If I set the value of "BrowserExecutableFolder" in the path SOFTWARE\Policies\Microsoft\MyApplication, can WebView2 pick up the value from this path?

**Comment by vickiez** [comment]
Hi @stevealex1, WebView2 will only pick up overrides under SOFTWARE\Policies\Microsoft\Edge\WebView2. Is there some reason you want to change the path?

**Comment by stevealex1** [comment]
"[WebView2-specific policies](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-webview-policies) are available for you to manage WebView2 directly. However, we recommend that WebView2 app developers implement their own group policies to manage the use of WebView2, because it's easier for administrators to manage the app instead of managing WebView2 directly."
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/enterprise.

This is the point we are trying to get implemented.
Could you please help us to implement our own group policies to manage the use of WebView2?

**Comment by vickiez** [comment]
Are you able to accomplish this using WebView2 APIs? i.e. develop your own policy, read the value in the app, and use WebView2 APIs to ensure that the WebView behaves according to the policy
